lib/web_ui/lib/src/engine/platform_dispatcher.dart

```diff
@@ -138,9 +138,8 @@ class EnginePlatformDispatcher extends ui.PlatformDispatcher {
 
   /// The current list of windows,
   @override
-  Iterable<ui.FlutterView> get views => _windows.values;
-  Map<Object, ui.FlutterWindow> get windows => _windows;
-  final Map<Object, ui.FlutterWindow> _windows = <Object, ui.FlutterWindow>{};
+  Iterable<ui.FlutterView> get views => viewData.values;
+  final Map<Object, ui.FlutterView> viewData = <Object, ui.FlutterView>{};
```

Comment:
I think the iteration order of a plain HashMap in Dart is not guaranteed, so `viewData` *must* be a `LinkedHashMap` if we want indices to be consistent through the `views` Iterable, right?

https://api.dart.dev/be/175791/dart-core/Map-class.html

Reply:
(Or is `views` a stopgap measure until we retrieve directly from the viewData by their ViewId?)

Note that this also exists in the mobile lib/ui/platform_dispatcher.dart (~line 165)

Reply:
map literals are LinkedHashmaps and the iteration order is gauranteed.

Reply:
(This doesn't seem to be [documented accurately in the Dart language tour](https://dart.dev/guides/language/language-tour#maps:~:text=Dart%20infers%20that%20gifts%20has%20the%20type%20Map%3CString%2C%20String%3E%20and%20nobleGases%20has%20the%20type%20Map%3Cint%2C%20String%3E). Adding @mit-mit to take a look and file an issue if necessary.)

Reply:
It's documented in the Map *constructor* [here](https://api.flutter.dev/flutter/dart-core/Map/Map.html), but I couldn't find that piece of lore looking at the main pages of Map/LinkedHashMap, apologies for the noise in the PR!!

Reply:
@lrhn promised to take a look at the API docs